src/components/Markdown.tsx

```diff
@@ -0,0 +1,16 @@
+import { BoxProps } from '@material-ui/core';
+import { marked } from 'marked';
+
+import CleanHtml from './CleanHtml';
+
+interface MarkdownProps {
+  BoxProps?: BoxProps;
+  markdown: string;
+}
+
+const Markdown = ({ BoxProps, markdown }: MarkdownProps): JSX.Element => {
```

Comment:
This could be written:

```tsx
const Markdown: React.FC<MarkdownProps> = ({ BoxProps, markdown}) => {
```

I think that's more readable (and includes slightly more functionality too, I believe, although that's not used here).